src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -67,7 +71,7 @@ case class Profile (
 object Profile {
 
   // increment this number every time you make a change to the user-provided profile fields
-  val currentVersion:Int = 2
+  val currentVersion:Int = 3
```

Comment:
It's a magic number that developers need to remember to increment manually when something else changes. I don't like remembering things and I don't like manual things. I'd prefer deriving a hash or something from the class definition itself.